command/ibft/quorum/params.go

```diff
@@ -0,0 +1,102 @@
+package quorum
+
+import (
+	"errors"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"os"
+)
+
+const (
+	fromFlag  = "from"
+	chainFlag = "chain"
+)
+
+var (
+	params = &quorumParams{}
+)
+
+type quorumParams struct {
+	genesisConfig *chain.Chain
+	from          uint64
+	genesisPath   string
+}
+
+func (p *quorumParams) initChain() error {
+	cc, err := chain.Import(p.genesisPath)
+	if err != nil {
+		return fmt.Errorf(
+			"failed to load chain config from %s: %w",
+			p.genesisPath,
+			err,
+		)
+	}
+
+	p.genesisConfig = cc
+
+	return nil
+}
+
+func (p *quorumParams) initRawParams() error {
+	if err := p.initChain(); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Just a suggestion,nothing important. Here we can just return p.initChain();

Reply:
Good catch

[Simplified](https://github.com/0xPolygon/polygon-edge/pull/549/commits/78ac602703bc607ed2e91bb5bec5ce1b65ff2631)